lib/ansible/plugins/__init__.py

```diff
@@ -415,8 +415,12 @@ def all(self, *args, **kwargs):
                 continue
 
             if path not in self._module_cache:
-                self._module_cache[path] = self._load_module_source(name, path)
-                found_in_cache = False
+                try:
+                    self._module_cache[path] = self._load_module_source(name, path)
```

Comment:
make this fatal by default, add option in 'all' call to make it 'permissive' of failures